frontend/src/stories/index.stories.js

```diff
@@ -0,0 +1,11 @@
+import {storiesOf} from '@storybook/react';
```

Comment:
__[ESLint]__ Error - Parsing error: The keyword 'import' is reserved ([link](https://sider.review/gh/repos/108558043/pulls/347#issue-4065188))

<sub>posted by Sider</sub>